File: restart_service/restart_service.go
package main

import (
	"fmt"
	"os/exec"
)

func main() {
	service := "CloudflareWARP" // Exact service name

	cmd := exec.Command("powershell", "Restart-Service", service)
	err := cmd.Run()
	if err != nil {
		fmt.Printf("[ERROR] Failed to restart %s: %v\n", service, err)
		return
	}

	fmt.Printf("[OK] %s restarted successfully.\n", service)
}

File: Python scripts/restart_service.py
import subprocess

# Use the exact service name
service_name = "CloudflareWARP"

try:
    # Restart the service
    subprocess.run(["powershell", "Restart-Service", service_name], check=True)
    print(f"[OK] {service_name} restarted successfully.")
except subprocess.CalledProcessError:
    print(f"[ERROR] Failed to restart {service_name}.")
